src/period-calculation/get-fixed-period-by-date/get-weekly-fixed-period-by-date.ts

```diff
@@ -21,8 +21,8 @@ const getWeeklyFixedPeriodByDate: GetWeeklyFixedPeriodByDate = ({
         startingDay: 1,
     })
 
-    // if start date of first period of year is after current date get last
-    // period of last year
+    // if current date is before start date of first period of year
```

Comment:
I just modified the language of the comment here, so that it more closely matched the actual logical operation performed below